app/src/main/java/com/sun/android/ex14/SimplePowerReceiverActivity.kt

```diff
@@ -0,0 +1,47 @@
+package com.sun.android.ex14
+
+import android.content.Intent
+import android.content.IntentFilter
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.localbroadcastmanager.content.LocalBroadcastManager
+import com.sun.android.databinding.ActivitySimplePowerReceiverBinding
+
+
+class SimplePowerReceiverActivity : AppCompatActivity() {
+    private val binding by lazy { ActivitySimplePowerReceiverBinding.inflate(layoutInflater) }
+
+    private val mSimpleReceiver by lazy { SimpleReceiver() }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+
+        setReceiver()
+
+        binding.buttonSendBroadcast.setOnClickListener(::sendBroadcast)
```

Comment:
```suggestion

        binding.buttonSendBroadcast.setOnClickListener { _ ->
                   sendBroadcast()
        }
```
bỏ tham số view ở sendBroadcast, hạn chế tham số không dùng nhá